.gitattributes

```diff
@@ -0,0 +1,2 @@
+/internal/website/data/examples.json  linguist-generated=true
+**/testdata/**/*.yaml  linguist-generated=true
```

Comment:
Please add `**/testdata/**/*.replay` as well. Some of them are binary, so are probably collapsed by default, but others are text (recorded HTTP requests/responses).

Reply:
Done.